modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -345,10 +323,18 @@ def _get_index_and_columns(df):
     int
         The number of columns.
     """
-    return len(df.index), len(df.columns)
+    logger = get_worker_logger(JOB_ID)
+    logger.info(
+        f"START::PANDAS-API::PandasOnRayDataframePartition._get_index_and_columns"
+    )
+    res = len(df.index), len(df.columns)
+    logger.info(
+        f"STOP::PANDAS-API::PandasOnRayDataframePartition._get_index_and_columns"
+    )
+    return res
```

Comment:
I would rather we implement specific worker logger decorator which would do this instead of copy-pasting this pattern all over the place... plus why `PANDAS-API`?..

Reply:
Added a decorator `enable_remote_logging` which does this.